contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+            <td>Push and pull data to other destinations</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Session replays:** [Session replays](/session-replay) in PostHog recreate exactly what real users see and how they use your product. They also enable you to debug problems using built-in [console logs](/docs/session-replay/configure#console-logs-recording-beta), [performance monitoring](/tutorials/performance-metrics) and the [DOM explorer](/blog/posthog-changelog#experimental-dom-explorer-mode).
+
+- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
+
+- **A/B testing:** In PostHog, you can use the [experimentation suite](/ab-testing) to create multivariate tests within your product, such as showing some users a different page layout to others. Over time, you can build an understanding of which page performs better, correlate results with other events, and deploy a final version.
+- 
+### Session replays
+
+Session replays are an essential tool for understanding how people use your product, especially for [early-stage companies](/blog/early-stage-analytics) searching for product-market fit. Both Hotjar and PostHog offer session replays:
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+			<td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Free recording allowance</strong></td>
+            <td className="text-center">~1k per month (max 35 per day) on Basic plan</td>
+            <td className="text-center">15k per month</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Price for first 50k sessions</strong></td>
+            <td className="text-center">~$200 per month (max 1.5k sessions per day)</td>
+            <td className="text-center">$175 per month</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Price for first 50k sessions</strong></td>
+            <td className="text-center">~$200 per month (max 1.5k sessions per day)</td>
+            <td className="text-center">$175 per month</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Browser app recording</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Watch real users use your product; diagnose bugs.</td>
+        </tr>
+        <tr>
+            <td><strong>Connected to product analytics</strong></td>
+            <td className="text-center">Requires additional integration</td>
```

Comment:
I'd just change this to a no. Hotjar doesn't do this; it requires an additional tool that does it.